Index.html

```diff
@@ -0,0 +1,313 @@
+                <h2 class="BackE">Back End dev</h2>
+                <img src="./images/Counter.png" alt="counter count" class="dot" />
+                <h2 class="date">2015</h2>
+              </div>
+              <div clas="adayca">
+                <p class="adayca_p">
+                  A daily selection of privately </br>
+                  personalized reads; no accounts or </br> 
+                  sign-ups required.
+                </p>
+              </div>
+              <ul class="botonic">
+                <li class="cahtml">html</li>
+                <li class="cacss">css</li>
+                <li class="caJava">JavaScript</li>
+              </ul>
+              <div class="buttonca">
+                <button type="submit" class="go-buttonca">See project</button>
+              </div>
+            </li>
+          </li>
+        </ul>
+        <ul class="flex4">
+          <li class="mult"></li>
+          <li class="square4">
+            <li class="primarytext4">
+              <h1 class="multi4">
+                Multi-post</br>
+                Stories
+              </h1>
+              <div class="cardtonic">
+                <h2 class="CANOPY">CANOPY</h2>
+                <img src="./images/Counter.png" alt="counter count" class="dot1" />
+                <h2 class="BackE">Back End dev</h2>
+                <img src="./images/Counter.png" alt="counter count" class="dot" />
+                <h2 class="date">2015</h2>
+              </div>
+              <div clas="aday4">
+                <p class="aday4_p">
+                  A daily selection of privately </br>
+                  personalized reads; no accounts or </br> 
+                  sign-ups required.
+                </p>
+              </div>
+              <ul class="botonic">
+                <li class="multhtml">html</li>
+                <li class="multcss">css</li>
+                <li class="multJava">JavaScript</li>
+              </ul>
+              <div class="buttonmult">
+                <button type="submit" class="go-buttonmult">See project</button>
+              </div>
+            </li>
+          </li>
+        </ul>
+      </nav>
+    </main>
+    <section class="contact-me">
+      <div class="head-section-co">
+        <div id="head-container-co">
+          <h2 class="head-co">
+            About <br />
+            Myself
+          </h2>
+          <p class="description-co">
+            Hello I’m a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h2 class="connect-co">LET’S CONNECT</h2>
+          <ul class="social-media-icons">
+            <li>
+              <a href="#" target="_blank">
+                <img src="images/github.png" alt="git" />
```

Comment:
- [ ] [OPTIONAL] This is an **OPTIONAL** changes, however, I **HIGHLY** recommend you to implement it because it would enhance the quality of your code 🤗 
Please add more horizontal spacing (**padding**) in this section. If you compare your design to the Figma template, you will notice discrepancy 🤖 
Kindly fix that to make your webpage beautiful 💅🏽 ❤️